source/unified-test-format/unified-test-format.rst

```diff
@@ -3,13 +3,13 @@ Unified Test Format
 ===================
 
 :Spec Title: Unified Test Format
-:Spec Version: 1.2.3
+:Spec Version: 1.3.0
```

Comment:
FYI: this and the corresponding schema version are going to conflict https://github.com/mongodb/specifications/pull/935. Please reach out to @divjotarora and decide if you should use 1.4.0 here or vice versa.

I'll note that the schema versioning doesn't force drivers to implement the projects in a particular order. IIRC, I was working on Atlas testing (1.2 with `loop`, etc.) in PHPLIB before we had implemented versioned API (1.1).

Reply:
Good point. I think whichever one is ready to merge first ought to take the 1.3 and the other can do 1.4. I'll check with him before merging anything to ensure we're on the same page.